index.js

```diff
@@ -0,0 +1,43 @@
+// const qs = require("qs")
+
+document.addEventListener("DOMContentLoad", () => {
+    console.log("hello")
+    // let welcomeBtn = document.querySelector("#welcomeBtn")
+
+
+
+    let classBtn = document.querySelector("#classBtn");
+    let studentBtn = document.querySelector("#studentBtn");
+    let fsBtn = document.querySelector("fsBtn");
+    let classDisplay = document.querySelector("#newClassDisplay")
+
+
+    let url = "http://localhost:3000/school"
+
+    const displayNewClass = (url) => {
+        
+    }
+
+
+
+
+
+
+    // addClassButton
+
+
+classBtn.addEventListener("submit", async (e) => {
+    e.preventDefault();
+    await axios.post(`http://localhost:3000/class/`, { name: className.value, teacher: teacher.value })
```

Comment:
You didn't grab your `className` and `teacher` inputs from your html file., so you are not sending anything in your request.